doc/source/dev/toolchain.rst

```diff
@@ -14,9 +14,9 @@ The tooling used to build CPython has some implications for the tooling used in
 Python Versions
 ---------------
 
-SciPy is compatible with several versions of Python.
-[*NOTE: The level of support and dates is currently under discussion.
-This table shows what it could look like.*]
+SciPy is compatible with several versions of Python, and some
+specific decisions are still under consideration, especially
+with regard to future changes.
```

Comment:
I'm afraid that this kind of language doesn't really help the reader decide anything. The previous italics were a reminder to update this with the right statement.